src/test/storage/storage_manager_test.cpp

```diff
@@ -27,6 +27,22 @@ TEST_F(StorageStorageManagerTest, GetTable) {
   EXPECT_THROW(sm.get_table("third_table"), std::exception);
 }
 
+TEST_F(StorageStorageManagerTest, TableNames) {
```

Comment:
StorageStorageManager is doubled. Maybe copy pasta?

Reply:
die heißen tatsächlich alle so...